cookbooks/bcpc-hadoop/recipes/graphite_queries.rb

```diff
@@ -72,17 +106,17 @@
         'route_to' => "admin"
       }
     end
-    
+
     if triggers[host]["#{disk}_space_used_#{host}"].nil?
       triggers[host]["#{disk}_space_used_#{host}"] = {
         'type' => "servers",
         'query' => "diskspace.#{disk}.byte_used",
         'trigger_val' => "max(61,0)",
-        'value_type' => 3,
-        'trigger_cond' => ">#{(size * 0.80).floor}G",
+        'value_type' => 0,
+        'trigger_cond' => ">#{(size * 0.90).round(2)}G",
         'trigger_name' => "#{host}_#{disk}_SpaceUsed",
         'enable' => true,
-        'trigger_desc' => "More than 80% of disk space used",
+        'trigger_desc' => "More than 90% of disk space used",
```

Comment:
Again the percentage is quite arbitrary and the total remaining bytes as well. The big question is how much does data usually grow over time and can we support some amount of time of growth (e.g. down to ~5 days of space left) is probably more important with some fall backs of minimum expected slop space (e.g. enough to run maintenance Map/Reduce tasks and such).

Again, for now this should be great.